### ISSUE TITLE
data model V1HTTPGetAction can not pass the API server side validation

### ISSUE THREAD

**Issue by joshua-bj** [kubernetes-client/java#128]
If the type of the V1HTTPGetAction.port should be changed to the IntOrString? If use the String type, when transform to the JSON, it always double quoted, which can not pass the API server's validation. I tried at the API server 1.7.4 and 1.8.2. It will failed with
`The Deployment "event-deployment-dev" is invalid: spec.template.spec.containers[0].livenessProbe.httpGet.port: Invalid value: "5454": must contain at least one letter or number (a-z, 0-9)`

**Comment by joshua-bj** [comment]
I tried to replace the String type to the IntOrString locally, the API invoke success now.
The client code change to:
```
 V1HTTPGetAction httpGet = new V1HTTPGetAction();
 httpGet.setPath("/");
 httpGet.setPort(new IntOrString(5454));
```

**Comment by joshua-bj** [comment]
it's already fixed. Thanks.

**Comment by rjeberhard** [comment]
What version has this fix?  I see it's fixed in master, but 1.0.0beta1 doesn't have the fix.
